.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Documentation
+about: Report an issue with our documentation
+labels: 'type: documentation, triage'
+assignees: ''
+
+---
+
+<!--
+  - Thanks for helping us improve our developer site documentation!
+  - Use this template to describe issues with the content at
+  - developers.google.com/blockly/guides
+  -->
+
+**Where**
+
+<!-- A link to the page with the documentation you want us to update.
+  -  More specific is better.  If no page exists, describe what the page
+  -  should be, and where.
+  -->
```

Comment:
It might be nice to add some checkboxes for 'What'
[ ] Text
[ ] Image or Gif
[ ] Other

Reply:
Seems redundant, since I'm asking them to put that text or gif in the issue as well.

Reply:
As is I read it only as asking for the text. I don't see anything that specifically asks them to note if we need to update an image.

Reply:
Okay, updating and I'll add it to my cherry-pick as well.